recipes/genesis2/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "genesis2" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/Genesis-1.3-Version2/archive/v{{ version }}.tar.gz
+  sha256: 0c8be1fd80cfa8339e3e40b5e798ec596d339e9b743c46a187321a71b9766e12
+  patches:
+    - size.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}                    # [not win]
+    - {{ compiler('m2w64_c') }}              # [win]
+    - {{ compiler('fortran') }}              # [not win]
+    - {{ compiler('m2w64_fortran') }}        # [win]
```

This shouldn't matter, but maybe it helps Conda recognize `build` and `host` are separate. Thinking maybe Conda is confused about our requirements and thus has issues installing this package into the test environment.

```suggestion
    - {{ compiler('m2w64_fortran') }}        # [win]
  host:
  run:
```